src/state/slices/portfolioSlice/portfolioSliceCommon.ts

```diff
@@ -54,6 +54,11 @@ export type ConnectWallet = {
 }
 
 export type Portfolio = {
+  /**
+   * The app can run account discovery taking a lot of time and slowing down the portfolio initialization,
+   * we use this key to gracefully handle this state in any part of the app
+   */
+  isAccountsLoading: boolean
```

Comment:
with either `isAccountMetadataLoading` or `isPortfolioLoading` the intention would be pretty clear without any need for comment imo

Reply:
Removed in [e72bded](https://github.com/shapeshift/web/pull/7629/commits/e72bded769397fe997646cd23ed7ff15a20d7bc6)